shaders/glsl/sun_and_moon.fragment

```diff
@@ -76,10 +76,15 @@ void main()
 	highp vec3 dp1 = dFdx(worldPosition.xyz);
 	highp vec3 dp2 = dFdy(worldPosition.xyz);
 
-	 vec3 normal = (normalize(cross(dp1, dp2)) + 1.0) / 2.0;
+	vec3 normal = (normalize(cross(dp1, dp2)) + 1.0) / 2.0;
 	
 
 	gl_FragColor = CURRENT_COLOR * diffuse * length(diffuse.rgb);
 	gl_FragColor += halo * haloTint /*+ sun * sunTint*/;
-	gl_FragColor.a *= 1.0 - isRain;
+
+	float sunHaloIntensity = 0.6;
+	float moonHaloIntensity = 0.4;
+	float haloIntensity = sunHaloIntensity + abs((isDay * moonHaloIntensity) - moonHaloIntensity);
+
+	gl_FragColor.a *= haloIntensity - (isRain * haloMultiplier);

```

Comment:
I can't find initialisation of haloMulitplier anywehere. Are you shure it works? And what is the reason to use haloIntencity  and haloMulitplier? May be it will be better to use only one of these two variables

Reply:
Oh crap that's a typo sorry 😅
They used to all used haloMultiplier but I changed it. Must have missed that one instance

Reply:
Fixed it

Reply:
plz pull ඞ